code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -250,20 +250,6 @@
 	else if(bodytemperature > maxbodytemp)
 		adjustBruteLoss(heat_damage_per_tick)
 
-/mob/living/simple_animal/Bumped(AM as mob|obj)

```

Comment:
This resolves the issues with simple animals sometimes teleporting when bumped into.